index.html

```diff
@@ -0,0 +1,169 @@
+                </a>
+            </li>
+            <li class="nav-text"> 
+                <a href="#" class="nav-link">
+                    <span class="text">iPad</span>
+                </a>
+            </li>
+            <li class="nav-text"> 
+                <a href="#" class="nav-link">
+                    <span class="text">itunes</span>
+                </a>
+            </li>
+            <li class="nav-text"> 
+                <a href="#" class="nav-link">
+                    <span class="text">Support</span>
+                </a>
+            </li>
+        </ul>
+        <div class="searchbox-wrapper">
+            <form class="form-container">
+                <div class="searchbox-container">
+                    <input type="text" name="searchbox" id="searchbox">
+                </div>
+            </form>
+        </div>
+        </nav>
+        <main>
+            <div class="grid-wrapper">
+                <section class="hero">
+                        <div class="grid-header">
+                            <a class="background-image">
+                            </a>         
+                            <header class='hero-header'>
+                                <h1 class="main-header">
+                                    <a>
+                                        <img class="head-img1" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_title_small.png" alt="What will your verse be?"/>
+                                    </a>
+                                </h1>
+                                <h2 class="secondary-header">
+                                    <a>
+                                        <img class="head-img2" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_subhead_small.png" alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story."/>
+                                    </a>
+                                </h2>
+                                <div class="watch-now">
+                                    <a class="watch-link">
+                                        <img class="head-img3" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/cta_your_verse_27pt.png" alt="Watch the “Your Verse” film and explore the stories"/>
+                                    </a>
+                                </div>
+                            </header>
+                        </div>    
+                </section>
+                <div class="grid-footer">
+                    <section class="info1 info-shaper">
+                            <a class="info-image">
+                                <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5s.jpg" alt="iPhone 5s. Forward thinking.">
+                            </a>
+                    </section>
```

Comment:
## Link tag
- [ ] Each `<a>` should an href `<a href="#">`

**Note:** change all `<a>` tags in this section